src/components/Content/SearchResult/MovieList/MovieList.module.css

```diff
@@ -0,0 +1,9 @@
+.ml_container {
+    display: grid;
```

Comment:
I think it is not the best idea to use grid and flex in the same application. At least I've never seen it before. I'd suggest to use one approach

Reply:
I agree with you but I want to cover  both CSS layout approaches, because I'm noob) and I'm not sure what approach more preferred for me.   Using CSS Grid is the simple way to create table view(rows/columns) , but for case with to objects are at the edges of page CSS Flex more usful for me.

Reply:
But of cause I can mistake:)

Reply:
If it is necessary for self-education - of course, you can use both approaches